src/com/lftechnology/java/training/sanish/application/controller/LoginController.java

```diff
@@ -0,0 +1,60 @@
+package com.lftechnology.java.training.sanish.application.controller;
+
+import com.lftechnology.java.training.sanish.application.component.Constants;
+import com.lftechnology.java.training.sanish.application.dbconnection.DbConnect;
+import com.lftechnology.java.training.sanish.application.component.UserLogin;
+import com.lftechnology.java.training.sanish.application.view.login.LoginPage;
+
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Login controller
+ *
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class LoginController {
+    private static final Logger LOGGER = Logger.getLogger(LoginController.class.getName());
+
+    private LoginController() {
+    }
+
+    /**
+     * Login page
+     *
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static void LoginPage() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00100)